libs/spark/src/Unstable_IconButton/Unstable_IconButton.stories.tsx

```diff
@@ -2,39 +2,34 @@ import React from 'react';
 import type { Meta, Story } from '@storybook/react/types-6-0';
 import { theme, Unstable_IconButton, Unstable_IconButtonProps } from '..';
 import {
-  Unstable_ChevronDown,
-  Unstable_Filter,
-  Unstable_Info,
-} from '../internal';
-import { containFocusIndicator, sparkThemeProvider } from '../../stories';
-
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-interface Sb_Unstable_IconButtonProps
-  extends Omit<Unstable_IconButtonProps, 'tabIndex'> {}
-
-export const Sb_Unstable_IconButton = (props: Sb_Unstable_IconButtonProps) => (
-  <Unstable_IconButton {...props} />
-);
+  ChevronDown,
+  containFocusIndicator,
+  Info,
+  Plus,
+  sparkThemeProvider,
+} from '../../stories';
+
+export const _retyped = Unstable_IconButton as typeof Unstable_IconButton;
 
 export default {
   title: '@ps/Unstable_IconButton',
-  component: Sb_Unstable_IconButton,
-  excludeStories: ['Sb_Unstable_IconButton'],
+  component: _retyped,
+  excludeStories: ['_retyped'],
   parameters: { actions: { argTypesRegex: '^on.*' } },
   decorators: [containFocusIndicator],
   argTypes: {
     children: {
       control: 'select',
-      options: ['Unstable_ChevronDown', 'Unstable_Filter', 'Unstable_Info'],
+      options: ['<ChevronDown />', '<Info />', '<Plus />'],
       mapping: {
-        Unstable_ChevronDown: <Unstable_ChevronDown />,
-        Unstable_Filter: <Unstable_Filter />,
-        Unstable_Info: <Unstable_Info />,
+        '<ChevronDown />': <ChevronDown />,
+        '<Info />': <Info />,
+        '<Plus />': <Plus />,
```

Comment:
I don't want to be pulling anything from the package's internal directory, only its main index. And we don't need to prefix storybook helpers with "(u|U)nstable_" because they're not exposed to consumers.